pros/src/sensors/rotation.rs

```diff
@@ -30,7 +31,13 @@ impl RotationSensor {
     }
 
     /// Sets the position to zero.
+    #[deprecate_until(remove = "1.x", note = "Please use the `set_zero` method instead.")]
     pub fn zero(&mut self) -> Result<(), PortError> {
+        self.set_zero()
+    }
+
+    /// Sets the position to zero.
+    pub fn set_zero(&mut self) -> Result<(), PortError> {
```

Comment:
Even though you might consider the name `zero` to be against common Rust conventions, I think that `set_zero` could easily be confused with `set_position`.

Reply:
Whilst I agree that the naming could be confused with setting the position, I do feel that it does clarify the meaning a bit more since the word zero is both a noun and a verb. I think it might be worthwhile in the long run to adopt either Vex's `reset` terminology or PROS's `tare` terminology, and have respective `reset`/`tare` and `reset_to_position`/`tare_to_position` functions on motors/sensors instead.

Reply:
I generally agree with gavin to the extent that "set_zero" is a more confusing name (could be misinterpreted as "set the zero point"). Although at the same time using something like reset would solve both of these problems.